docs/metrics.md

```diff
@@ -6,188 +6,46 @@
-| [Tomcat JDBC connection pool metrics](#connection-pool-metrics)      | `tomcat-jdbc-splunk`   | 8.5 and higher
-| [Vibur DBCP connection pool metrics](#connection-pool-metrics)       | `vibur-dbcp-splunk`    | 20.0 and higher
 | [Tomcat thread pool metrics](#thread-pool-metrics)                   | `tomcat`               | 8.5 and higher
 | [WebSphere Liberty web request thread pool](#thread-pool-metrics)    | `liberty`              | 20.0.0.12
 | [WebLogic thread pools](#thread-pool-metrics)                        | `weblogic`             | 12.x and 14.x
 
 ### JVM
 
-We use the [built-in Micrometer JVM metrics extension](https://micrometer.io/docs/ref/jvm)
-to register JVM measurements.
-
-#### Classloader metrics
-
-| Metric name                    | Instrument   | Description |
-| ------------------------------ | ------------ | ----------- |
-| `runtime.jvm.classes.loaded`   | [Gauge][g]   | The number of loaded classes.
-| `runtime.jvm.classes.unloaded` | [Counter][c] | The total number of unloaded classes since the process started.
-
-#### GC metrics
-
-| Metric name                            | Instrument   | Description |
-| -------------------------------------- | ------------ | ----------- |
-| `runtime.jvm.gc.concurrent.phase.time` | [Timer][t]   | Time spent in concurrent phase, in milliseconds.
-| `runtime.jvm.gc.live.data.size`        | [Gauge][g]   | Size of long-lived heap memory pool after reclamation, in bytes.
-| `runtime.jvm.gc.max.data.size`         | [Gauge][g]   | Max size of long-lived heap memory pool, in bytes.
-| `runtime.jvm.gc.memory.allocated`      | [Counter][c] | Incremented for an increase in the size of the young heap memory pool after one garbage collection to before the next.
-| `runtime.jvm.gc.memory.promoted`       | [Counter][c] | Count of positive increases in the size of the old generation memory pool before garbage collection to after garbage collection.
-| `runtime.jvm.gc.pause`                 | [Timer][t]   | Time spent in garbage collection pause, in milliseconds.
-
-#### Heap pressure metrics
+We use [OpenTelemetry JVM Metrics](https://opentelemetry.io/docs/specs/semconv/runtime/jvm-metrics/).
+Besides OpenTelemetry JVM Metrics we include the following JVM metrics.
 
-| Metric name                         | Instrument   | Description |
-|-------------------------------------| ------------ | ----------- |
-| `runtime.jvm.gc.overhead`           | [Gauge][g]   | An approximation of the percentage of CPU time used by GC activities over the last lookback period or since monitoring began, whichever is shorter, in the range [0..1].
-| `runtime.jvm.memory.usage.after.gc` | [Gauge][g]   | The percentage of long-lived heap pool used after the last GC event, in the range [0..1].
+#### Memory profiler metrics
 
-#### Memory metrics
+These metrics are enabled only when memory profiler is enabled.
```

Comment:
```suggestion
These metrics are only enabled when the memory profiler is enabled or when setting `splunk.metrics.enabled=true`.
```

Reply:
Setting `splunk.metrics.enabled=true` is not enough to enable all of them. See https://github.com/signalfx/splunk-otel-java/blob/3099b48ee6c890495022098a4c6c31b9dea369ae/instrumentation/jvm-metrics/src/main/java/com/splunk/opentelemetry/instrumentation/jvmmetrics/JvmMetricsInstaller.java#L59-L66 there is another flag that is needed to enable most of them (as far as I remember we use that flag for testing).